code/controllers/subsystem/verb_manager.dm

```diff
@@ -24,7 +24,7 @@ SUBSYSTEM_DEF(verb_manager)
 	wait = 1
 	flags = SS_TICKER | SS_NO_INIT
 	priority = FIRE_PRIORITY_DELAYED_VERBS
-	runlevels = RUNLEVEL_INIT | RUNLEVELS_DEFAULT
+	runlevels = RUNLEVEL_LOBBY | RUNLEVELS_DEFAULT
```

Comment:
I need someone to comment on the correctness of this

Reply:
i need you to remove the define so i can see that file from this pr and check something to answer ont his.

Reply:
oh wait i see, ya this is correct. verb manager should have been running in the lobby.